data_steward/analytics/cdr_ops/controlled_tier_qc/check_controlled_tier_part2.py

```diff
@@ -1385,27 +1385,62 @@ def query_template(table_era):
 GROUP BY value_source_value
 ORDER BY value_source_value
 """)
-q = query.render(
-    project_id=project_id,
-    ct_dataset=ct_dataset,
+q = query.render(project_id=project_id,
+                 ct_dataset=ct_dataset,
 )
 result = execute(client, q)
 if not result.empty:
     df = df.append(
         {
-            'query': 'Query19 race/ethnicity sub-categories exist.',
-            'result': 'PASS',
+            'query': 'Query19 existence of at least one race/ethnicity sub-categories',
+            'result': 'PASS'
         },
         ignore_index=True)
 else:
     df = df.append(
         {
-            'query': 'Query19 race/ethnicity sub-categories DO NOT exist.',
-            'result': 'Failure',
+            'query': 'Query19 At least one race/ethnicity sub-categories DOES NOT exist',
+            'result': 'Failure'
         },
         ignore_index=True)
 result
 
+# Query 20: verify all race/ethnicity values are top-level, not subcategories
+# -
+
+query = JINJA_ENV.from_string("""
+SELECT ethnicity_concept_id, ethnicity_source_concept_id, ethnicity_source_value, count(*)
```

Comment:
This query would not print the results from the 2nd query after union all as the column names are different. You would have to change the query structure to a CTE and then combine results from both the sub queries into a different format.